### DISCUSSION TITLE
Announcing the v2.36.5 Release

### DISCUSSION THREAD

**Discussion by mapkon** [Versent/saml2aws#997] [Announcements]
Hello folks!

After a long period of coding and maintenance, I am pleased to announce the v2.36.5 release. There has been a tonne of changes in this release, which can be seen [here](https://github.com/Versent/saml2aws/compare/v2.36.4...v2.36.5). This release mainly focused on:

- Bumping the versions of dependencies (via dependabot)
- Add `authentik` as an authentication provider
- Improving the CI process by adding codecov as a code coverage tool
- Bumping the dependencies and versions used in CI pipeline
- Adding tests for existing functionality
- Fixing google apps login
- Allow Symantec VIP to be passed as param for OTP method for okta signin
- Fix Duo authentication failure with Shib provider
- Execute command directly without sh -c
- And many more ...

At the time of writing, the latest binaries have not yet made it to the relevant repositories like homebrew, but should be reflected there in a few hours!

I would like to extend special thanks to the contributors, specifically:

- @gliptak 
- @wolfeidau 
- @briantist 
- @jwdinkel 
- @rwstauner 
- @chris-reeves 
- @charltona 
- @falms 
- @smlx 
- @ttitoo 

If I have missed out anyone, it is not intentional. 

Moving forward, we are aiming to move to a more regular release cadence, with small releases over a couple of weeks in order to avoid the big bang release like this one. The discussion is on-going [here](https://github.com/Versent/saml2aws/discussions/995) - and we would love your thoughts/inputs there.

Thank you

**Comment by charltona** [comment]
Thanks @mapkon!
